_posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Adopting Docs-as-Code: From Hackathon to Production"
+excerpt: Sysdig's journey from unstructured to structured, finally to semi-structured authoring and how Sysdig hackathon enabled design their homegrown docs-like-code solution.
+last_modified_at: 
+categories: articles
```

Comment:
Let's try the `canonical_url: "https://sysdig.com/blog/adopting-docs-as-code/"` line here in the front matter and see if the Jekyll SEO plugin works like I think it should.